runtime/src/lib.rs

```diff
@@ -329,6 +329,19 @@ impl pallet_sudo::Config for Runtime {
     type WeightInfo = weights::pallet_sudo::NHWeight<Runtime>;
 }
 
+parameter_types! {
+    pub const MaxSignatories: u32 = 100;
+}
+impl pallet_multisig::Config for Runtime {
+    type RuntimeEvent = RuntimeEvent;
+    type RuntimeCall = RuntimeCall;
+    type Currency = Balances;
+    type DepositBase = ConstU128<EXISTENTIAL_DEPOSIT>;
+    type DepositFactor = ConstU128<0>;
```

Comment:
To be discussed: 
DepositFactor -> The amount of currency needed per unit threshold when creating a multisig execution. Recall: The deposit to be made by the account that creates the multisig is: `threshold * DepositFactor + DepositBase` 
Set as 0 for now